src/doc/src/reference/source-replacement.md

```diff
@@ -20,6 +20,12 @@ like so:
 [source.my-awesome-source]
 directory = "vendor"
 
+# Git sources can optionally specify a branch/tag/rev as well
```

Comment:
5b08b8f added this entry after “Available kinds of sources that can be specified” in the “Configuration” section around line 71.  This commit adds the entry to an example in the top “Replacing sources” section.  I think we need to shift this entry down.